detect_secrets_server/actions/scan.py

```diff
@@ -26,6 +27,10 @@ def scan_repo(args):
         log.error('Unable to find repo: %s', args.repo)
         return 1
 
+    # if last_commit_hash is empty, go ahead and try to reinitalize to see if there's a new base hash
```

Comment:
What do you mean by base hash?

Nit: `reinitialize`

Reply:
modified comment language to be more clear. `base hash` should have been `first commit hash`